packages/app/components/Sidebar/SidebarMenu.tsx

```diff
@@ -1,5 +1,6 @@
 'use client'
 
+import { BookOpenText } from 'lucide-react'
```

Comment:
Please use react-icons/lu instead of lucide-react

Reply:
sorry, didn't check that you are actually using the `lu` icons I'm used to `lucide` but it is true that they are going to get rid of it or something, i saw it somewhere hehe.

Reply:
`react-icons/lu` is `lucide`. The package name is just different